doc/reference/dune-project/dialect.rst

```diff
@@ -66,6 +66,17 @@ dialect
 
       .. versionchanged:: 3.9 This field is made optional.
 
+   .. describe:: (merlin_reader <program> <args>...)
+
+      Configure Merlin to use `<program> <args>...` as READER. Merlin's READER
+      is a mechanism to extend Merlin to support OCaml dialects by providing
+      a program that transforms a dialect AST into an OCaml AST. See `merlin/src/extend/extend_protocol.ml <https://github.com/ocaml/merlin/blob/master/src/extend/extend_protocol.ml>`_
+      for the protocol specification.
```

Comment:
Can you make that a link to a stable version (tag instead of `master`)? Also, maybe a `seealso` block would work well for this (see above).